### PR TITLE
Port & Implement Assets page (- ibc transfer)

### PR DESCRIPTION
**Pull request by jonator**
Closes: #242 

* Added the osmosis account at `@osmosis-labs/stores/assets` but did not add to root store at `root.ts`
  * Includes message composition code
* Refactored IBC transfer logic, but did not implement (due to problem above)
* Tweak some components, namely pool-card and search-box
* Wrapped some `@osmosis-labs/stores` objects with an observable mobx class: `ObservableAssets` (at `packages/web/stores/assets` 
  * Pros: globally accessible memoized calculations 
  * Cons: Organization/architecture doesn't seem fit. Was considering writing a hook with `useMemo` called `useUserAssets` or the like, but wouldn't get the pro above.

### CODE REVIEW

File: packages/web/components/control/menu-dropdown.tsx
Comment (delivan) [inline packages/web/components/control/menu-dropdown.tsx:41]: `nanoid` for React component's key is not recommended. Check out the [docs](https://github.com/ai/nanoid#react)

File: packages/web/components/input/input-box.tsx
Comment (delivan) [inline packages/web/components/input/input-box.tsx:84]: nanoid for React component's key is not recommended.

**Comment by jonator** [comment]
Converting to draft to implement updated IBC transfer dialog design/UI

File: packages/stores/package.json
Comment (jonator) [reply packages/stores/package.json]: For some of the `@keplr-wallet/stores` imports I needed to use `^0.9.16` to avoid build errors.

File: packages/stores/src/account/msg/make-msg.ts
Comment (jonator) [reply packages/stores/src/account/msg/make-msg.ts:74]: My goal with this is to keep primitive types to pass to the math package. That reduces the dependency on constructing these objects when not necessary. Also, `CoinPretty` doesn't expose amount as public (it's protected).

File: packages/web/components/cards/pool-card.tsx
Comment (jonator) [reply packages/web/components/cards/pool-card.tsx:14]: We can wait until the pools PR is merged before merging this PR.

**Comment by jonator** [comment]
This PR is blocked by PR #287.

**Comment by jonator** [comment]
@delivan I merged to master and reconciled some conflicts. Notable files to review:
* packages/web/components/cards/pool-card.tsx
* packages/web/components/table/index.tsx
* packages/web/components/control/checkbox.tsx

Thanks!

**Review by delivan — COMMENTED** [review]
If I try to sort in order of network in sorting menu, I get the following error:
스크린샷 2022-03-25 오후 5 57 35

File: packages/web/hooks/use-ibc-transfer/types.ts
Comment (delivan) [inline packages/web/hooks/use-ibc-transfer/types.ts:18]: Remove the comments

File: packages/web/modals/ibc-transfer.tsx
Comment (delivan) [inline packages/web/modals/ibc-transfer.tsx:154]: Please allow to input dot for decimals

File: packages/web/components/table/assets-table.tsx
Comment (delivan) [inline packages/web/components/table/assets-table.tsx:229]: It would be better to remove hover styling to make the user aware nothing happened if clicking table row.

File: packages/web/pages/assets/index.tsx
Comment (delivan) [inline packages/web/pages/assets/index.tsx:217]: How about using `maxDecimals(2)` to sync with pool card on pools page?

**Review by delivan — CHANGES_REQUESTED** [review]

File: packages/web/pages/assets/index.tsx
Comment (jonator) [reply packages/web/pages/assets/index.tsx:216]: @osmosis-labs/data victor and I agree that the info site should replace the rotating OSMO logo with simple geometric placeholders. I think syncing design here would very much help with consistent branding across app and info.

Screen Shot 2022-03-28 at 1 40 50 PM

**Comment by jonator** [comment]
Yet to do:

- [ ] Create additional custom withdrawal safeguard: [Figma](https://www.figma.com/proto/7iuO8MZeiYezTnF7suwTNN/draft-UI?page-id=106%3A6758&node-id=476%3A30575&viewport=332%2C48%2C0.25&scaling=min-zoom&starting-point-node-id=476%3A30575)

**Comment by delivan** [comment]
LGTM besides withdrawal safeguard 👍

**Comment by jonator** [comment]
@delivan 

I've added the new store init logic here, and added the withdraw safeguard. Feel free to approve. 🚀

**Review by delivan — APPROVED** [review]

**Pull request merged**
